grr/server/grr_response_server/gui/ui/components/timestamp/timestamp.ts

```diff
@@ -12,5 +12,6 @@ export class Timestamp {
   @Input() date?: Date;
   @Input() completeFormat: boolean = false;
   @Input() tooltipEnabled: boolean = true;
+  @Input() disableUnderlining: boolean = false;
```

Comment:
I get your point and I agree in general, but this case is a bit tricky, so let me share my thought process and figure out what is best:

Whether the text will be underlined depends on [multiple things](https://github.com/simstoykov/grr/blob/simeon-first-task/grr/server/grr_response_server/gui/ui/components/timestamp/timestamp.ng.html#L7). In the form the current expression is, if we change the variable name to "underline" we might confuse the user of the component that by setting "underline" to true, the text will always be underlined, which is not the case - `completeFormat` should be false and `tooltipEnabled` should be true in addition to `underline` being true. That's why I went for "disableUnderlining", hoping that it's more explicit that if `disableUnderlining` is false, then the user is still not guaranteed to get underlining --- it's just not disabled. Maybe `forceDisableUnderlining` is even more explicit (but longer and even further away from your suggestion). 

Maybe the cleanest thing to do long term would be to remove [this underlining logic](https://github.com/simstoykov/grr/blob/simeon-first-task/grr/server/grr_response_server/gui/ui/components/timestamp/timestamp.ng.html#L7) and only control the underlining with an explicit `underline` variable, as you suggested. This would require going through the other 9 uses of `<timestamp>` and making sure that the behavior is not changed there.

What do you think?

Reply:
Good reasoning. I realize that underlining is intended to indicate "more information on hover", but is missing in most of the design pages. I suggest you follow a simple approach of your choice and we possibly remove the underline alltogether after talking with Melinda.

Reply:
I would go with `forceDisableUnderlining` at this point because it doesn't require modifying other code (which might create conflicts) and it's the most explicit one for the time being.

Reply:
SG